app/models/room.rb

```diff
@@ -17,29 +17,19 @@ class Room < ApplicationRecord
   extend FriendlyId
   friendly_id :name, use: :scoped, scope: :space
 
+  ACCESS_LEVELS = %i[internal public].freeze
   # A Room's Access Level indicates what a participant must know in order to gain access to the room.
-  # `unlocked` indicates that the participant does not need to know anything to gain access.
-  # `locked` indicates that only participants who know the rrooms `access_code` may access the room.
-  # `internal` indicates that only participants who are Space Members _or_ know the Spaces `access_code` may
-  # access the room.
-  attribute :access_level, :string, default: "unlocked"
-
-  # A room's Access Code is a "secret" that, when known, grants access to the room.
-  attribute :access_code, :string
-  validates :access_code, presence: {if: :locked?}
-
-  def locked?
-    access_level&.to_sym == :locked
-  end
-
-  def unlocked?
-    access_level&.to_sym == :unlocked
-  end
+  # `internal` only Members may access the Room
+  attribute :access_level, :string, default: :public
```

Comment:
Not from this PR, and not blocking, but I'm surprised this is not an enum. It would save us from defining the `internal?` and `public?` methods, and also help make sure we don't accidentally set it to something that is not in the enum (like, say, during a refactor...).

```ruby
enum :access_level, {
  internal: "internal",
  public: "public"
}, default: :public
```

Then our one use of `Room::ACCESS_LEVELS.map(&:to_s)` becomes instead `Room.access_levels.values`.

Reply:
I considered piling that into this PR, but dissuaded myself because of the weirdness I ran into with the ENUMs on Heroku last time.